### PR TITLE
Made Pill component's  iconImageView property public

### PR DESCRIPTION
**Pull request by DivyaMandyamTT**
- Removed logic that sets iconSize in Config as this doesn't actually work to change the icon size.
- Made the `iconImageView` property public so that subclasses can access it and customize its layout/size.

### CODE REVIEW

File: Thumbprint/Targets/Thumbprint/Components/Pill.swift [hunk 1]
```diff
@@ -15,27 +15,23 @@ open class Pill: UIView, UIContentSizeCategoryAdjusting {
...
             self.padding = padding
             self.textStyle = textStyle
-            self.iconSize = iconSize
         }
     }
 
     // MARK: - Implementation Views
 
-    private let iconImageView: UIImageView = {
+    public let iconImageView: UIImageView = {
```
Comment (tt-pkaminski) [inline Thumbprint/Targets/Thumbprint/Components/Pill.swift:34]: QQ - are we exposing the UIImageView (as opposed to the UIImage) to allow the caller to change the positioning / constraints, etc?

Comment (DivyaMandyamTT) [reply Thumbprint/Targets/Thumbprint/Components/Pill.swift:34]: Yes

File: Thumbprint/Targets/Thumbprint/Components/Pill.swift [hunk 2]
```diff
@@ -235,8 +231,8 @@ open class Pill: UIView, UIContentSizeCategoryAdjusting {
         stackView.addArrangedSubview(label)
 
         iconImageView.snp.makeConstraints { make in
-            make.height.equalTo(config.iconSize.dimension)
-            make.width.equalTo(config.iconSize.dimension).priority(.high)
+            make.height.equalTo(Icon.Size.tiny.dimension)
+            make.width.equalTo(Icon.Size.tiny.dimension).priority(.high)
```
Comment (DivyaMandyamTT) [inline Thumbprint/Targets/Thumbprint/Components/Pill.swift:235]: Restored old logic for making the icon size tiny.

**Review by tt-pkaminski — APPROVED** [review]
LGTM

**Pull request merged**